### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```
Comment (vasylhnatiuk) [inline .gitignore]: add newline

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: cinema/models.py [hunk 2]
```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self) -> str:
+        return self.first_name + " " + self.last_name
```
Comment (ArsenPidhoretskyi) [inline cinema/models.py]: use f-string

File: cinema/urls.py [hunk 3]
```diff
@@ -1 +1,19 @@
...
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+router.register("actors", ActorViewSet)
+router.register("genres", GenreViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+urlpatterns = [
+    path("", include(router.urls))
+]
```
Comment (ArsenPidhoretskyi) [inline cinema/urls.py]: No need to include router when a basepath is not provided.
Use urlpatterns = router.urls

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: cinema/serializers.py [hunk 4]
```diff
@@ -1 +1,73 @@
...
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
```
Comment (viktoria-rybenchuk) [inline cinema/serializers.py]: Adding the trailing comma can help prevent potential syntax errors and makes it easier to modify the list by adding or removing elements

Comment (viktoria-rybenchuk) [reply cinema/serializers.py]: Fix in all places

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]
Fix flake8

**Review by pavlejviki — APPROVED** [review]
GJ!

**Pull request closed without merging**